crates/holochain/tests/publish/mod.rs

```diff
@@ -30,37 +31,31 @@ async fn publish_termination() {
 
     let ((alice,), (bobbo,), (carol,), (danny,), (emma,), (fred,)) = apps.into_tuples();
 
+    // trigger init and await consistency
+    let _: InitCallbackResult = conductors[0]
+        .call(&alice.zome(TestWasm::Create), "init", ())
+        .await;
+    consistency_60s([&alice, &bobbo, &carol, &danny, &emma, &fred]).await;
+
     let _: ActionHash = conductors[0]
         .call(&alice.zome(TestWasm::Create), "create_entry", ())
         .await;
 
     // Wait until they all see the created entry, at that point validation receipts should be getting sent soon
     consistency_60s([&alice, &bobbo, &carol, &danny, &emma, &fred]).await;
 
-    let ops_to_publish = tokio::time::timeout(Duration::from_secs(60), async {
-        let alice_pub_key = alice.agent_pubkey().clone();
-        loop {
-            tokio::time::sleep(Duration::from_millis(100)).await;
```

Comment:
Nothing happens while waiting here, because the validation receipt workflow isn't triggered unless the integration workflow had some changes. Since consistency has been reached before this loop, no other op will trigger these workflows.